02_bubble/mohankrishna/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprint(out, bubble([]int{3, 2, 1, 5}))
+}
+
+func bubble(s []int) (sorted []int) {
+	sorted = make([]int, len(s))
+	copy(sorted, s)
+	swapped := true
+	for swapped {
+		swapped = false
+		for i := 0; i < len(sorted)-1; i++ {
+			if sorted[i+1] < sorted[i] {
+				swapped = true
+				sorted[i+1], sorted[i] = sorted[i], sorted[i+1]
+			}
+		}
+	}
+	return
```

Comment:
Do we need return here as this is last line?

Reply:
Yes, it is mandatory.